### DISCUSSION TITLE
Code notes should be editable in RAWeb

### DISCUSSION THREAD

**Discussion by wescopeland** [RetroAchievements/RAWeb#1382] [Ideas]
Currently, notes and memory addresses are read-only. It would be useful if code notes could be edited and deleted from within RAWeb itself.

**Additional context**
Keep in mind the code notes table renders on multiple pages. On the achievement inspector page, it renders on the right-hand column with a very restricted width.

**Comment by Ryudo300** [comment]
100% agree.

It'd be a huge QoL for devs to have the ability to add/edit/delete code notes on the site itself, rather than having to do it 1 by 1 in the emulators. 

Especially with the new local code notes system in place in emulators, DELETING code notes is a tiring task where you have to first empty out the notes 1 by 1, and then publish them afterwards as well. Quite a bunch of extra tasks that could be made simpler if we were able to do it from the site.

tl;dr - Possible features:

- Select individual code notes
- Select all from x author
- Delete selected code notes
- Edit selected code notes (maybe handy for if you want to give the same description to multiple addresses).
- Delete all code notes from a game's page
- Add code notes
- Edit existing code notes (individual ones)

**Comment by wescopeland** [comment]
The ability to edit existing code notes is covered by this PR: https://github.com/RetroAchievements/RAWeb/pull/1440
